### ISSUE TITLE
Consider releasing as separate modules

### ISSUE THREAD

**Issue by tomekwi** [1-liners/1-liners#127]
Here’s an interesting discussion I took part in: <https://github.com/mattdesl/module-best-practices/pull/2>.

**Comment by stoeffel** [comment]
I'm totally :+1: on this. I think I discussed this with @hemanth some time ago.

**Comment by hemanth** [comment]
Yes, as it was with `fp-dom` and `fp-js` we need to restart that as well :wink:

**Comment by guumaster** [comment]
Are the `1-liners` a superset of `fp-js` repos?

**Comment by hemanth** [comment]
Well, kina. `fp-js` was made for all the supporting `libs` for `fp-dom`, later on `1-liners` picked up, it's your choice if you want to do a `fp-*` module as well.

**Comment by guumaster** [comment]
It may be redundant to have an `fp-*` container. Having `1-liners` with auto-curry options as discussed in #90 would be a better choice.

**Comment by stoeffel** [comment]
> Having 1-liners with auto-curry options as discussed in #90 would be a better choice.

:+1: We could also think about using 1-liners in fp-dom instead of fp-js.

**Comment by guumaster** [comment]
@stoeffel I've just tried that today, but the interfaces are not the same. So, I ended up here and found out about the differences between the projects. 

It doesn't seem so hard to adpat all of `fp-dom` to `1-liners`, but if the auto-curry option is not so far in the roadmap, i'd rather wait for it. Is that feature coming soon?

**Comment by stoeffel** [comment]
@guumaster I hope I can begin with the auto-curry feature soon. But if you are interested in starting it, contribution is always welcome.
